main.go

```diff
@@ -0,0 +1,448 @@
+				fileButton = widget.NewButton("Select file", func() {
+					fileDialog.Show()
+				})
+				form := widget.NewForm(&widget.FormItem{Text: "Path to word JSON file", Widget: fileButton},
+					&widget.FormItem{Text: "Player count", Widget: playerCount})
+				form.OnCancel = win.Close
+				form.OnSubmit = func() {
+					w, err := loadJsonFile(strings.TrimSpace(path))
+					if err != nil {
+						e := dialog.NewError(err, win)
+						e.Show()
+						return
+					}
+					playerCountInt, err := strconv.Atoi(playerCount.Text)
+					if err != nil {
+						e := dialog.NewError(errors.New("please enter a valid integer"), win)
+						e.Show()
+						return
+					}
+					if playerCountInt < 3 {
+						e := dialog.NewError(errors.New("player count cannot be less than 3, please try again"), win)
+						e.Show()
+						return
+					}
+					content.RemoveAll()
+					result = FakeDrawerInBrnoGUI(w, playerCountInt, win)
+				}
+				content.Add(form)
+			}),
+
+			widget.NewButtonWithIcon("Add words to JSON file", theme.ContentAddIcon(), func() {
+				content.RemoveAll()
+				var path string
+				var fileButton *widget.Button
+				fileButton = widget.NewButton("Select file", func() {
+					fileDialog := dialog.NewFileOpen(func(uc fyne.URIReadCloser, err error) {
+						if err != nil || uc == nil {
+							return
+						}
+						fileButton.SetText("File selected!")
+						path = uc.URI().Path()
+					}, win)
+					fileDialog.SetFilter(storage.NewExtensionFileFilter([]string{".json"}))
+					fileDialog.Show()
+				})
+				form := widget.NewForm(&widget.FormItem{
+					Text:   "Please select JSON file to edit",
+					Widget: fileButton})
+				form.OnCancel = win.Close
+				form.OnSubmit = func() {
+					result = addWordsToJSONGUI(path, win)
+				}
+				content.Add(form)
+			}),
+
+			widget.NewButtonWithIcon("Game help", theme.HelpIcon(), func() {
+				info := dialog.NewInformation("How to use",
+					"This is a game moderator for the board game A fake artist goes to New York.\nThe concept of the game is that out of all players there is one"+
+						" fake artist and the rest of the players are normal artists.\nAll players get a word category and only the normal artists get"+
+						" the actual word they have to draw.\nThen all players take turns drawing maximum one line on a shared piece of paper twice.\nThat is"+
+						" there will be two rounds of drawing one line. After these two rounds finish\neverybody votes on who they think is the fake. Then the fake"+
+						" reveals themselves and guesses what the word was.\nThe fake wins if they aren't guessed by the other players or if they guess the correct word."+
+						"\n\nThis app uses JSON files to load the words for the game. \nTo play the game or add words to a json file, you will be prompted\n"+
+						"to select a fitting JSON file.",
+					win)
+				info.Show()
+			}),
+
+			widget.NewButtonWithIcon("Quit", theme.CancelIcon(), app.Quit)))
+	win.SetContent(content)
+	win.ShowAndRun()
+	return result
+}
```

Comment:
Ideally, do not nest these. Separate them to different components that get combined here. This is pretty cluttered and unreadable.